sklearn/linear_model/ransac.py

```diff
@@ -243,6 +247,17 @@ def fit(self, X, y):
         except ValueError:
             pass
 
+        estimator_fit_has_sample_weight = has_fit_parameter(base_estimator, "sample_weight")
+        estimator_name = type(base_estimator).__name__
+        if (sample_weight is not None
+                    and not estimator_fit_has_sample_weight ):
+            raise ValueError("%s does not support sample_weight. Samples"
+                              " weights are only used for the calibration"
+                              " itself." % estimator_name)
```

Comment:
please fix the PEP8 stuff here. Here are the violations I get by running `pep8 sklearn/linear_model/ransac.py`

`sklearn/linear_model/ransac.py:253:21: E127 continuation line over-indented for visual indent`
`sklearn/linear_model/ransac.py:253:60: E202 whitespace before )`
`sklearn/linear_model/ransac.py:255:31: E127 continuation line over-indented for visual indent`